ProcessLib/StaggeredCouplingTerm.h

```diff
@@ -35,7 +35,22 @@ struct StaggeredCouplingTerm
     std::map<ProcessType, GlobalVector const*> const& coupled_xs;
 };
 
+struct LocalCouplingTerm
+{
+    LocalCouplingTerm(
+        std::map<ProcessType, Process const&> const& coupled_processes_,
+        std::map<ProcessType, const std::vector<double>>&& local_coupled_xs_)
+    : coupled_processes(coupled_processes_),
+      local_coupled_xs(std::move(local_coupled_xs_))
+    {
+    }
+
+    std::map<ProcessType, Process const&> const& coupled_processes;
+    std::map<ProcessType, const std::vector<double>> const local_coupled_xs;
+};
+
 const StaggeredCouplingTerm createVoidStaggeredCouplingTerm();
+const LocalCouplingTerm createVoidLocalCouplingTerm();
```

Comment:
It means an empty instance of LocalCouplingTerm. This function has already been dropped in another commit.